doc/Customization-guide/Documentation-contribution-and-style-guide/index.md

```diff
@@ -0,0 +1,245 @@
+
+- [Documentation style guide](#documentation-style-guide)
+  - [Sentences and grammar](#sentences-and-grammar)
+  - [Word choice](#word-choice)
+  - [Headings and capitalization](#headings-and-capitalization)
+  - [Technical terms and abbreviations](#technical-terms-and-abbreviations)
+  - [Font conventions](#font-conventions)
+  - [Code formatting](#code-formatting)
+  - [Page structure](#page-structure)
+  - [Markdown formatting](#markdown-formatting)
+  - [Jekyll formatting](#jekyll-formatting)
+- [Documentation contribution](#documentation-contribution)
+  - [Documentation preview](#documentation-preview)
+
+This guide introduces documentation conventions applicable to the Kaa project. The intent of this guide is to unify the look and feel of the documentation, make navigation predictable, usage and contribution simple.
+
+# Documentation style guide
+
+## Sentences and grammar
+* Use consistent terminology, so that every term means the same thing and is spelled in the same way throughout the entire documentation. If not properly introduced, synonyms can confuse the reader.
+* Give preference to present tense over past and future tenses; give preference to simple verb forms over perfect and continuous forms.
+* Use numbered lists for step-by-step procedures and bulleted list for an unordered set of items. Capitalize the first word of each list entry.
+* Use parallel sentence structures in bulleted and numbered lists.
+* End each entry of the list with a period if all the entries are complete sentences or if they are a mixture of fragments and complete sentences. If all entries are fragments, do not end them with periods or any other punctuation mark.
+* Use imperative mood in procedures, that is, formulate your instructions as giving commands to the user.
+* Avoid inventing new words or assigning words an unusual meaning.
+
+## Word choice
+* Avoid using please in instructions unless some step in the procedure causes inconvenience for the user or represents a workaround for some system limitation.
+
+## Headings and capitalization
+* Capitalize only the first word in titles and headings.
+* Do not use capitalization for no apparent reason. Use lowercase unless uppercase is justified.
+* Use Heading 4 (e.g. ####) as the last level heading.
+
+## Technical terms and abbreviations
+* Spell out a technical abbreviation on its first mention on the page.
+* Ensure that an important technical term or abbreviation is included in  [Glossary]({{{root_url}}Glossary).
+* Use *italic* formatting to emphasize a new technical term on its first mention.
+
+## Font conventions
+* Use **bold** for the titles of windows and dialog boxes; for the names of commands, attributes, constants, methods, fields, predefined classes, databases, events, UI elements; for the user input.
+* Use *italic* for terms on first mention and emphasis; for placeholders and parameter names; for section names.
```

Comment:
The part "Use *italic* for terms on first mention and emphasis" repeats the last item above (see "Technical terms and abbreviations"). Can this be just a link to it?